tree/ntuple/v7/src/RNTupleModel.cxx

```diff
@@ -43,9 +43,9 @@ ROOT::Experimental::RNTupleModel::RNTupleModel()
 ROOT::Experimental::RNTupleModel* ROOT::Experimental::RNTupleModel::Clone()
 {
    auto cloneModel = new RNTupleModel();
-   auto cloneFieldZero = static_cast<RFieldZero*>(fFieldZero->Clone(""));
-   cloneModel->fFieldZero = std::unique_ptr<RFieldZero>(cloneFieldZero);
-   cloneModel->fDefaultEntry = std::unique_ptr<REntry>(cloneFieldZero->GenerateEntry());
+   auto cloneFieldZero = fFieldZero->Clone("");
+   cloneModel->fFieldZero = std::unique_ptr<RFieldZero>(static_cast<RFieldZero *>(cloneFieldZero.release()));
+   cloneModel->fDefaultEntry = std::unique_ptr<REntry>(cloneModel->fFieldZero->GenerateEntry());
```

Comment:
Shouldn't GenerateEntry also return a std::unique_ptr and then use the same pattern as the line before (which ever we settle on)?

Reply:
Yep, addressed in #6655